_website/download/prelude.md

```diff
@@ -75,6 +75,25 @@ maintain backward compatibility from version to version.
 
 # OS-Specific Packages
 
+## Arch Linux
+
+Elvish PKGBUILDs are available in AUR. Install [`elvish`](https://aur.archlinux.org/packages/elvish/) or [`elvish-git` (HEAD)](https://aur.archlinux.org/packages/elvish-git/) using your favorite AUR helper:
+
+```elvish
+# Install latest release
+yay -S elvish
+# Or install HEAD
+yay -S elvish-git
+```
+
+Alternatively, pre-built packages can be obtained from [Arch Linux CN repository](https://www.archlinuxcn.org/archlinux-cn-repo-and-mirror/):
+
+```elvish
+echo -e '[archlinuxcn]\nServer = http://repo.archlinuxcn.org/$arch' | sudo tee -a /etc/pacman.conf
+sudo pacman -Sy
```

Comment:
IIRC, you can combine the two commands into `sudo pacman -Sy elvish` (please test whether it works!)